### DISCUSSION TITLE
作者是凌晨在更新么

### DISCUSSION THREAD

**Discussion by 8bytes-code** [walinejs/waline#1063] [Q&A] [answered]
半夜写博客本地预览的时候发现waline挂了。。。控制台报错说waline未定义。。当时想着用的明明是官网的js文件，难道是更新了，就去看了看，一开始看着还是用的cdn那个，vercel上测试的也挂了。就先去重新部署了下vercel上的服务，发现重新部署能看到了，版本也变成2.5.1了。。我印象里之前还是1.6.x来着，然后又去官网刷新了一下。。发现script的引用链接都变成unpkg了，还多了个link的css样式。。还好我之前找到了我主题的ejs文件，照着模板改了一下，就都行了，之前差点忘了css预览的时候没样式看着奇丑。

然后一点多又刷新了一下。。看官网的又变成1.6了，引用又变成cdn了。。。编辑日期变成三月份了，我就寻思作者是不是真的在凌晨更新啊哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈。后面刷了几次确定是变成2.5.x了就没管了。

哦对了，官网的好像还是2.5.2，我自己的是2.5.1，是因为官网的算测试版本吗

**Comment by lizheming** [comment, marked as answer]
官网文档本地有缓存，你这个应该是碰到缓存更新了。jsdelivr 的 CDN 地址有问题，可以换成 unpkg 的或者其他 CDN 地址

**Comment by Mister-Hope** [comment]
2.0 都发一个月了，看来是你自己的 CDN 缓存比较持久。
